modules/ROOT/pages/munit-extensions-maven-plugin-configuration.adoc

```diff
@@ -0,0 +1,534 @@
+= Configuring the MUnit Extensions Maven Plugin
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+The MUnit Extensions Maven plugin provides testing capabilities for extension development. Learn how to configure the MUnit Extensions Maven plugin to test your modules. 
+
+== Specify Additional Argument Lines
+
+Because you run the tests in a new JVM, you can specify additional argument lines to the JVM. Specify each argument in a separate `argLine`.
+
+[source,xml,linenums]
+----
+<plugins>
+  <plugin>
+    <groupId>com.mulesoft.munit</groupId>
+    <artifactId>munit-extensions-maven-plugin</artifactId>
+    <configuration>
+      ...
+      <argLines>
+          <argLine>-XX:MaxPermSize=512m</argLine>
+          <argLine>-Xmx1024m</argLine>
+      </argLines>
+      ...
+    </configuration>
+  </plugin>
+</plugins>
+----
+
+== Enable Runtime Discovery
+
+Runtime discovery looks for all Mule runtime versions published on Nexus to run
+against your tests.
+
+To enable runtime discovery, add the following configuration to the MUnit Extensions Maven plugin:
+
+[source,xml,linenums]
+----
+<plugins>
+  <plugin>
+    <groupId>com.mulesoft.munit.tools</groupId>
+    <artifactId>munit-extensions-maven-plugin</artifactId>
+    <configuration>
+      ...
+      <runtimeConfiguration>
+          <discoverRuntimes>
+              <product>CE</product> //<1>
+          </discoverRuntimes>
+      </runtimeConfiguration>
+      ...
+    </configuration>
+  </plugin>
+</plugins>
+----
+[calloutlist]
+.. This enables the runtime discovery feature for all released CE runtimes.
+
+From the command line, the property is `discoverRuntimes.product`.
+
+NOTE: The `product` can be CE (Community Edition), EE (Enterprise Edition), or ALL (both CE and EE).
+
+=== Include Snapshots
+
+To include snapshot versions of Mule runtime, add the following line to the runtime configuration:
+
+[source,xml,linenums]
+----
+<runtimeConfiguration>
+    <discoverRuntimes>
+        <product>CE</product>
+        <includeSnapshots>true</includeSnapshots>
+    </discoverRuntimes>
+</runtimeConfiguration>
+----
+
+From the command line, the property is `discoverRuntimes.includeSnapshots`. The default value is `false`.
+
+=== Discover Only the Latest Patches
+
+To discover only the latest patch versions of Mule runtime, add the following line to the runtime configuration:
+
+[source,xml,linenums]
+----
+<runtimeConfiguration>
+    <discoverRuntimes>
+        <product>CE</product>
+        <latestPatches>true</latestPatches>
+    </discoverRuntimes>
+</runtimeConfiguration>
+----
+
+From the command line, the property is `discoverRuntimes.latestPatches`. The default value is `true`.
+
+=== Discover Since a Minimum Version
+
+By default, runtime discovery searches for all Mule runtime versions since the `minMuleVersion` of the module. To change this minimum version, add the following line to the runtime configuration:
```

Comment:
```suggestion
By default, runtime discovery searches for all Mule runtime versions since the `minMuleVersion` of the module. To change the minimum version, add the following line to the runtime configuration:
```